fvm/executionParameters.go

```diff
@@ -216,15 +211,6 @@ func (computer ExecutionParametersComputer) getExecutionParameters() (
 		return overrides, err
 	}
 
-	executionVersion, err := GetMinimumRequiredExecutionVersion(computer.log, computer.ctx, env)
```

Comment:
Just to confirm, this is the key change to remove the dependency on the execution state in order to prevent program cache to invalidated, right? 

I wonder why program cache would be invalidated if the operation is a read (reading the minimum required execution version), instead of a write.